exporters/trace/README.md

```diff
@@ -1 +1,51 @@
 # Cloud Trace Exporter for OpenTelemetry
+
+  OpenTelemetry Google Cloud Trace Exporter allows the user to send collected traces to Google Cloud. 
+  
+  <a href="https://cloud.google.com/trace">Google Cloud Trace</a> is a distributed tracing backend system. It helps developers to gather timing data needed to troubleshoot latency problems in microservice & monolithic architectures. It manages both the collection and lookup of gathered trace data.
+
+## Setup
+  Google Cloud Trace is a managed service provided by Google Cloud Platform.
+
+## Installation
+  Not currently ready as a package to be imported. To use presently, one would have to clone this GitHub repo.
```

Comment:
should we have the "to use presently". We're not even 100% it works yet

Reply:
Sounds good, I linked to issue#6 and put a line of code in there for cloning the repo